supabase/migrations/20250624123032_fix_spotify_url_case.sql

```diff
@@ -0,0 +1,24 @@
+-- Fix case sensitivity for Spotify URLs
+
+CREATE OR REPLACE FUNCTION clean_socials_profile_url()
+RETURNS TRIGGER AS $$
+BEGIN
+  -- Strip protocol + www (case-insensitive)
+  NEW.profile_url := regexp_replace(
+    NEW.profile_url,
+    '^(https?://)?(www\.)+',
+    '',
+    'i'
+  );
+
+  -- Remove trailing slashes
+  NEW.profile_url := regexp_replace(NEW.profile_url, '/+$', '');
+
+  -- Lower-case everything unless it starts with Spotify
+  IF NEW.profile_url !~* '^(open\.)?spotify\.com/' THEN
```

Comment:
why do you need to check `open`?
- why not simplify the refex to only `spotify`?
- you could event remove .com suffix
- principle: simple is best